src/core/nodeOps.ts

```diff
@@ -272,6 +272,11 @@ export const nodeOps: () => RendererOptions<TresObject, TresObject | null> = ()
           if (Array.isArray(value)) { node[finalKey](...value) }
           else { node[finalKey](value) }
         }
+        // NOTE: Set on* callbacks
+        // Issue: https://github.com/Tresjs/tres/issues/360
+        if (finalKey.startsWith('on') && isFunction(value)) {
```

Comment:
Thanks! Will take a look.

Reply:
Hey @garrlker , 

Have you been able to trigger this? I can't reproduce. 

I've checked with `@` events, as in the events playground – the step debugger never arrives there. Is there another case of events handlers that need to be guarded against?

## Context

`nodeOps` is a bit crufty at the moment. `supportedPointerEvents` is from the previous events implementation, I believe. 

@alvarosabu , `supportedPointerEvents` in `nodeOps` is still going away, right? Or did I misunderstand?

## Moving forward?

No problem moving the code into the `if` above. 

During writing I left it outside because as far as I know, the predicate is cruft. Adding lines inside the `if` means more uncertainty for the person who removes the cruft.

But I'll follow whatever you guys think.

Reply:
> I've checked with @ events, e.g., stepping through nodeOps on the new events playground. Events like @click/on-click don't reach this branch of patchProp anyway.

Oh good! Thanks for checking that out

> Is there another case of event handlers that need to be guarded against?

No, the pointer events were the only case I could think of possibly clashing here. If you've checked and the events aren't reaching this case then I'm good with the changes

And you're right, if we're going to remove that supportedEvents array then you shouldn't have to worry about it either way